### DISCUSSION TITLE
shrine/

### DISCUSSION THREAD

**Discussion by giscus** [lyricalpaws/NebSite#3] [General]
# shrine/

A shrine deep in a bamboo forest, belonging to a certain Kitsune

https://neb.gay/shrine/

**Comment by lyricalpaws** [comment]
![Testing fox](https://i.imgur.com/KrduK1q.png)

**Comment by patcolyn** [comment]
I leave this at the shrine, a gift of limitless giddiness.
Click between the eyes and see into its infinite wisdom.

[![Link to video of an owl](https://img.youtube.com/vi/jdtoHEN6ukU/0.jpg)](https://youtube.com/shorts/jdtoHEN6ukU)
